docker-compose.yml

```diff
@@ -1,17 +1,38 @@
-version: "3"
+version: "3.7"
 
 services:
-  docusaurus:
-    build: .
+  web:
+    image: bitcoinscala/wallet-server-ui:1.9.0-9ca96ac1-SNAPSHOT
+    user: "1000:1000"
```

Comment:
If i change this to `root` and below in the `walletserver` service to `root` this error no longer occurs: 

https://github.com/bitcoin-s/bitcoin-s/pull/4218#issuecomment-1077923564

I'm following this answer on SO: https://stackoverflow.com/questions/48727548/how-to-configure-docker-compose-yml-to-up-a-container-as-root